uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -482,17 +482,12 @@ private static function validateWorkingDayAndBalanceChange($params, $absenceType
    * @return float
    */
   private static function calculateBalanceChangeFromCreateParams($params) {
-    $leaveRequestOptionsValue = self::getLeaveRequestDayTypeOptionsGroupByValue();
```

Comment:
you are not using getLeaveRequestDayTypeOptionsGroupByValue method any more , shouldn't you remove it ?